generators/app/index.babel.js

```diff
@@ -94,7 +94,7 @@ export default Base.extend({
       this.getPackageVersions(
         'deps',
         [
-          'babel-core',
+          ['babel-core', '5.0.33']
```

Comment:
@kaylanm  just for future reference you can use `5.0.0` for all the babel deps and it will install the latest in the 5.x.x range without needing to look up exact versions. It will function exactly the same as you have it here, but just a tip to make it easier.